connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/DistributedConfig.java

```diff
@@ -67,6 +67,20 @@
             " fails to catch up within worker.sync.timeout.ms, leave the Connect cluster for this long before rejoining.";
     public static final int WORKER_UNSYNC_BACKOFF_MS_DEFAULT = 5 * 60 * 1000;
 
+    /**
+     * <code>offset.storage.topic</code>
+     */
+    public static final String OFFSET_STORAGE_TOPIC_CONFIG = "offset.storage.topic";
```

Comment:
The changes in these configs looks good, but the options still aren't used *outside* the config classes. A good way to be sure we've switched everything over to use the configs parsed by this class is to make sure there are no other cases where the config names (i.e. "offset.storage.topic", "config.storage.topic", and "offset.storage.file.filename") do not exist anywhere else in the code anymore.